autopilot/contractor.go

```diff
@@ -1422,7 +1422,8 @@ func (c *contractor) renewContract(ctx context.Context, w Worker, ci contractInf
 			"renterFunds", renterFunds,
 			"expectedNewStorage", expectedNewStorage,
 		)
-		if strings.Contains(err.Error(), wallet.ErrInsufficientBalance.Error()) {
+		if strings.Contains(err.Error(), wallet.ErrInsufficientBalance.Error()) &&
```

Comment:
we have `isErr` in the `contractor` so we can get rid of `strings.Contains` here